src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -331,7 +331,7 @@ internal void SendInvocationRequest(ScriptInvocationContext context)
                     };
                     foreach (var pair in context.BindingData)
                     {
-                        if (pair.Value != null)
+                        if (pair.Value != null && !pair.Key.Contains("request") && !pair.Key.Equals("req", StringComparison.OrdinalIgnoreCase))
```

Comment:
Contains doesn't look correct here. What's your intent?

Reply:
Synced up offline with @yojagad . 'req' is a parameter name and not keyword

Reply:
Basically, any bindings of type "DefaultHttpRequest" need to be ignored (since it's already in the input). Hope the updated change makes this clearer.